tools/dotnet-linker/Steps/ManagedRegistrarStep.cs

```diff
@@ -166,6 +166,8 @@ protected override void TryProcessAssembly (AssemblyDefinition assembly)
 				var md = abr.RegistrarHelper_RuntimeTypeHandleEquals.Resolve ();
 				md.IsPublic = true;
 				Annotations.Mark (md);
+			} else if (App.XamarinRuntime == XamarinRuntime.NativeAOT && Configuration.Profile.IsProductAssembly (assembly)) {
+				ImplementNSObjectRegisterToggleRefMethodStub ();
```

Comment:
This doesn't really fit with the managed static registrar, but I think it's good for now.

At some point we might want to move random stuff out of the managed static registrar and into a "MakeEverythingWorkWithNativeAOTStep"...